README

```diff
@@ -322,8 +322,14 @@
     OpenPOWER Host OS repositories on GitHub:
       https://github.com/open-power-host-os
 
+8. Pre/Post Script support 
 
-8. KNOWN ISSUES AND LIMITATION
+  User need to popluate their pre/post scripts as  prescript and postscript directory
+  * Create a directory named "prescript" in current directory and populated with pre scripts
```

Comment:
after we moved it to config directory this needs a change too.

Reply:
@sathnaga ah missed that, now rectified it